.travis.yml

```diff
@@ -24,7 +24,7 @@ install:
   - pip install -r ../requirements.txt
   - sudo apt-get install coreutils libxml2-utils
 script:
-  - pre-commit run --all-files
   - ./run-all-conversions
+  - pre-commit run --all-files
```

Comment:
I assume you want to run tests on all the generated files as well ? Good idea :).

Reply:
Yes. Subworkflow converter requires a properties..